react/features/base/conference/actions.js

```diff
@@ -359,6 +360,26 @@ export function _setAudioOnlyVideoMuted(muted: boolean) {
 }
 
 /**
+ * Action to set whether or not the currently displayed large video is in
+ * high-definition. Will no-op if there is no update to the known resolution.
+ *
+ * @param {boolean} isLargeVideoHD - True if the large video is high-definition.
+ * @returns {Function}
+ */
+export function setLargeVideoHDStatus(isLargeVideoHD) {
+    return (dispatch, getState) => {
+        const conference = getState()['features/base/conference'];
+
+        if (conference.isLargeVideoHD !== isLargeVideoHD) {
```

Comment:
You are absolutely correct. I swear I wrote a comment there explaining why but looks like I got rid of it. So even I knew it was a special case. That check is basically me being overly concerned with not trigging the redux switch cases more often than needed. In the current implementation of large video, setLargeVideoHDStatus gets called with every LOCAL_STATS_UPDATED. However, it's probably fine to get rid of the if-statement, especially if leaving it in means introducing a pattern that people might needlessly copy. The better fix is probably more upstream anyway, like throttling the stats updated listener.